contracts/core/RebalancingSetToken.sol

```diff
@@ -411,9 +411,9 @@ contract RebalancingSetToken is
         external
         returns (address[], uint256[], uint256[])
     {
-        // Make sure sender is Core
+        // Make sure sender is rebalanceAuctionModule
         require(
-            msg.sender == IRebalancingSetFactory(factory).core(),
+            msg.sender == IRebalancingSetFactory(factory).rebalanceAuctionModule(),
```

Comment:
Main reason I didn't add specific module names to the Core is because when we add new modules we won't be able to add their specific identifier to Core. As I think about it we don't actually need to specifically identify which module is calling these functions just that a module we approve of is, so I'd be fine just doing something like `ICore(core).validModule(msg.sender)` which I think accomplishes the same thing instead of specifying the module we need.

Reply:
+1